File: src/main/java/com/kerosenelabs/billtracker/model/response/GetIntroductorySettingsResponse.java
package com.kerosenelabs.billtracker.model.response;

import com.kerosenelabs.billtracker.model.IntroductorySettings;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class GetIntroductorySettingsResponse {
    private IntroductorySettings introductorySettings;
}

File: src/main/java/com/kerosenelabs/billtracker/model/response/GetIntroductorySettingsResponse.kt
package com.kerosenelabs.billtracker.model.response

import com.kerosenelabs.billtracker.model.IntroductorySettings
import lombok.AllArgsConstructor
import lombok.Data
import lombok.NoArgsConstructor

@Data
@AllArgsConstructor
@NoArgsConstructor
class GetIntroductorySettingsResponse {
    private var introductorySettings: IntroductorySettings? = null
}
